hw2/03-count.html

```diff
@@ -6,19 +6,20 @@
     <title>03 - Count</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="03-count.js"></script>
   </head>
-  <body>
-    <main>
+  <body class="row bg-dark justify-content-center">
```

Comment:
The background color is not supposed to be black.
It's supposed to be some kind of dark-bluish color.

Reply:
Yeah, I had noticed this but hadn't wanted to add extra styling just for that. I have fixed it though, I just used color picker.